_posts/2016-03-07-shaping-an-open-source-editorial-team.md

```diff
@@ -0,0 +1,72 @@
+---
+
+title: 'Shaping an open source editorial team'
+author: Jenn
+layout: blog-post
+comments: false
+
+---
+
+<img src="/blog/images/201603/community.jpg" alt="Community" style="border: 0">
+
+First thing–if you’re reading this and you know an awesome writer friend, share it with them.
+
+Second, if you’re a writer who managed to find their way here, welcome! You’re probably wondering “What is an online Hoodie?” and/or “What the heck does Open Source mean?”
+
+Hoodie is a group of fun folks who created a bit of code and decided to make it available to anyone with an internet connection and a sense of adventure. Instead of keeping their code to themselves, and limiting its capabilities to just a certain set of brains, they opened it up to the entire internet (and world) so that endless contributors can help shape and grow all the different ways Hoodie can now be useful.
+
+That approach, instead of a more proprietary one, is what people in the tech industry call “Open Source.”
+
+The folks at Hoodie are a little different though, did I mention adventure? They know that there’s a whole world outside of this tech bubble, where folks with millions of different talents help make amazing things every day. So why would they want to limit their open source community to just the kind of contributor that knows how to write computer code?
+
+In comes the __Hoodie editorial team__. The editorial team began as a [discussion in October 2015](https://github.com/hoodiehq/discussion/issues/82). The original thought was to first, recruit members of the team, who could then be responsible for planning and writing blog posts on or around Hoodie, Hoodie releases, contributors, side projects, etc. There was also some mention of coordinating Twitter and helping with support, but we’ll get to that.
+
+After much social media-ing and direct messaging, the editorial team officially started on the 21st of November, 2015 with an awesome Google Hangout full of people. Coincidentally, on the same day as a New York City-based [meetup for people getting familiarized with Hoodie](http://hood.ie/blog/meet-the-hoodies-offline.html).
+
+Soon after, the team created [a Slack channel](http://hood.ie/chat/) and [GitHub repository](https://github.com/hoodiehq/editorial/). GitHub is a social coding site that offers a free storage and collaboration service to customers, which can be conveniently used to store documents, hold discussions, and draft copy, among other things. Think Google Docs, for programmers.
```

Comment:
> service to customers,

maybe work in that it is free to use for open source projects, so people don’t get discouraged signing up